tests/unit/s2n_self_talk_client_hello_cb_test.c

```diff
@@ -104,6 +105,22 @@ int mock_client(struct s2n_test_io_pair *io_pair, int expect_failure, int expect
     exit(result);
 }
 
+bool security_policy_contains_cipher(const char* security_policy_name, uint8_t* cipher_iana) {
```

Comment:
Nice, it looks like this is working with the FIPS test now. The only failing test is for clang-format:
```suggestion
bool security_policy_contains_cipher(const char *security_policy_name, uint8_t *cipher_iana)
{
```